plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -168,21 +170,25 @@ public void checkVersionSpecificUpdateCenter() {
 
 
     public void writeFailedPluginsToFile() {
-
-        try (FileWriter fileWriter = new FileWriter("failedplugins.txt")) {
-            if (failedPlugins.size() > 0) {
-                System.out.println("Some plugins failed to download: ");
-                for (Plugin plugin : failedPlugins) {
-                    String failedPluginName = plugin.getName();
-                    System.out.println(failedPluginName);
-                    fileWriter.write(failedPluginName + "\n");
+        try (FileOutputStream fileOutputStream = new FileOutputStream("failedplugins.txt")) {
```

Comment:
would this be simpler?:
```suggestion
        try (
          FileOutputStream fileOutputStream = new FileOutputStream("failedplugins.txt");
          Writer fstream = new OutputStreamWriter(fileOutputStream, StandardCharsets.UTF_8)
        ) {
```